modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -119,25 +121,25 @@ If you don't see the *Flows* tab, verify that you are using Unified Agent and ha
 If you don't see the *Connectors* tab, verify that you have a Titanium subscription to Anypoint Platform and that the app you selected is a hybrid app deployed to a server, server group, or cluster. See xref:monitor-connectors.adoc[Monitor Connectors].
 | *Performance* a|
 
-* Average response time - Inbound
-* Average response time grouped by endpoint - Inbound
-* Mule messages
-* Response time graphs: 99, 90, 75, and 50 percentile inbound
-* Average response time  - Outbound
-* Average response time grouped by endpoint - Outbound
-* Response time graphs: 99, 90, 75, and 50 percentile - Outbound
+* Average Response Time - Inbound
+* Average Response Time Grouped by Endpoint - Inbound
+* Mule Messages
+* Response Time graphs: 99, 90, 75, and 50 percentile inbound
+* Average Response Time  - Outbound
+* Average Response Time Grouped by Endpoint - Outbound
+* Response Time graphs: 99, 90, 75, and 50 percentile - Outbound
 //* Response Time graphs: 99, 90, 75, and 50 Percentile Inbound
 //* Average Response Time: Outbound, Grouped by Endpoint Outbound
 | *Failures* a|
 
-* Total failed inbound requests
-* Total failed outbound requests
-* All inbound requests grouped by response type
-* All outbound requests grouped by response type
-* Failed inbound requests: Response time by endpoint
-* Total failed inbound requests grouped by endpoint
-* Failed outbound requests: Response time by HTTP endpoint (only HTTP endpoints are shown)
-* Total failed outbound requests grouped by HTTP endpoint (only HTTP endpoints are shown)
+* Total Failed Inbound Requests
+* Total Failed Outbound Requests
+* All Inbound Requests grouped by response type
+* All Outbound Requests grouped by response type
+* Failed Inbound Requests: Response time by endpoint
+* Total Failed Inbound Requests Grouped by Endpoint
```

Comment:
For consistency with other entries, "Grouped by Endpoint"  should be lowercase unless it is part of the name.

Reply:
```suggestion
* Total Failed Inbound Requests grouped by endpoint
```